lib/structures/VoiceState.ts

```diff
@@ -40,7 +40,6 @@ export default class VoiceState extends Base {
     user: User;
     constructor(data: RawVoiceState, client: Client) {
         super(data.user_id, client);
-        this.channel = null;
```

Comment:
Removing this causes typescript to complain the property has no initializer, why are you doing it?